DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+%% LyX 2.2.3 created this file.  For more info, see http://www.lyx.org/.
+%% Do not edit unless you really know what you are doing.
+\documentclass[DM,lsstdraft,toc,authoryear]{lsstdoc}
+\ifx\hypersetup\undefined
+  \AtBeginDocument{%
+    \hypersetup{unicode=true}
+  }
+\else
+  \hypersetup{unicode=true}
+\fi
+
+\makeatletter
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LyX specific LaTeX commands.
+
+\title[jointcal]{jointcal: Simultaneous Astrometry \& Photometry for thousands of
+Exposures with Large CCD Mosaics}
+\author{John Parejko (University of Washington), Pierre Astier (LPNHE/IN2P3/CNRS
+Paris)}
+\setDocRef{DMTN-036}
+\date{\today}
+\setDocAbstract{The jointcal package simultaneously optimizes the astrometric and
+photometric calibrations of a set of astronomical images. In principle
+and often in practice, this approach produces distortion and thoroughput
+models which are more precise than when fitted independently. This
+is especially true when the images are deeper than the astrometric
+reference catalogs. In the ``Astromatic'' software suite, this simultaneous
+astrometry functionality is fulfilled by ``SCAMP''. The code we
+describe here has similar aims, but follows a slightly different route.
+Jointcal is built on top of the the LSST Data Management software
+stack.}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% User specified LaTeX commands.
+\newcommand{\ClName}[1]{{\bf #1}}
```

Comment:
Please live in the 1990s in a world where latex2e exists.

Reply:
I have no idea what you mean by that. The `newcommand` blocks here were in the original document from Pierre. What should I use instead?

Reply:
`\textbf` not `{\bf`

Reply:
I've removed both of those commands and just used the `\code{}` and `\method{}` commands from lsst-texmf.